<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>联系人与群预览</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/4.png</normaloff>../images/4.png</iconset>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/newPrefix/images/8.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/newPrefix/images/3.jpg);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/newPrefix/images/7.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Gothic Std B</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>94</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>消息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>联系人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>群</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/6.png</normaloff>../images/6.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>391</width>
     <height>561</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="设置"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>780</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string extracomment="设置">设置</string>
   </property>
   <property name="accessibleDescription">
    <string extracomment="设置">设置</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/9.png</normaloff>../images/9.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>780</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="设置"/>
   </property>
   <property name="accessibleName">
    <string extracomment="添加好友">添加好友</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/10.png</normaloff>../images/10.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>Chat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>AddContacts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>788</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>820</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>SetSomething()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>799</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>823</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>ShowMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>818</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>ShowContacts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>874</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>ShowGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>876</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>ShowSearchMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>890</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Chat()</slot>
  <slot>AddContacts()</slot>
  <slot>SetSomething()</slot>
  <slot>ShowMessage()</slot>
  <slot>ShowContacts()</slot>
  <slot>ShowGroup()</slot>
  <slot>ShowSearchMessage()</slot>
 </slots>
</ui>
